lib/fathead/ruby/parse.rb

```diff
@@ -60,6 +60,10 @@ def to_s
     [to_row, to_redirect_rows].join
   end
 
+  def breaks_category_page?
+    title.include?('[') || abstract.include?('[')
+  end
```

Comment:
This flags up articles containing brackets, which currently breaks the category pages (see https://github.com/duckduckgo/zeroclickinfo-fathead/pull/277#issuecomment-237232714), and should be removed when that bug is fixed.